.github/workflows/api-update.yaml

```diff
@@ -1,8 +1,7 @@
 name: Update SDK
 on:
-  ## External event triggered by Repository
-  repository_dispatch:
-    types: [openapi-spec-change]
+  schedule:
+    - cron: '* * */1 * *'
```

Comment:
this is currently set to every minute? this might be a little over kill 😛

Reply:
Oops I thought this was every day I'll change